extensions/subscriptions/js/src/forum/addSubscriptionSettings.tsx

```diff
@@ -23,5 +24,20 @@ export default function () {
         app.translator.trans('flarum-subscriptions.forum.settings.follow_after_reply_label')
       )
     );
+
+    items.add(
+      'fof-subscriptions__notify-for-all-posts',
+      <FieldSet label={app.translator.trans('flarum-subscriptions.forum.settings.preferences_heading')} className="Settings-subscriptions">
+        <Switch
+          id="flarum_subscriptions__notify_for_all_posts"
+          state={!!this.user!.preferences()?.['flarum-subscriptions.notify_for_all_posts']}
+          onchange={(val: boolean) => {
+            this.user!.savePreferences({ 'flarum-subscriptions.notify_for_all_posts': val });
+          }}
+        >
+          {app.translator.trans('flarum-subscriptions.forum.settings.notify_for_all_posts_label')}
+        </Switch>
+      </FieldSet>
+    );
```

Comment:
I'd remove the heading, it doesn't really match well

```suggestion

    items.add(
      'notifyForAllPosts',
      <Switch
        id="flarum_subscriptions__notify_for_all_posts"
        state={!!this.user!.preferences()?.['flarum-subscriptions.notify_for_all_posts']}
        onchange={(val: boolean) => {
          this.user!.savePreferences({ 'flarum-subscriptions.notify_for_all_posts': val });
        }}
      >
        {app.translator.trans('flarum-subscriptions.forum.settings.notify_for_all_posts_label')}
      </Switch>
    );
```

![Screenshot from 2022-08-30 19-21-37](https://user-images.githubusercontent.com/20267363/187514125-00199616-0848-4208-b8c1-01ab5004c5aa.png)